imagersite/imager_images/tests.py

```diff
@@ -0,0 +1,77 @@
+from __future__ import unicode_literals
+from django.test import TestCase
+import factory
+from faker import Factory
+from . import models
+from .models import Photo, Album
+from django.contrib.auth.models import User
+
+fake = Factory.create()
+
+
+class UserFactory(factory.Factory):
+    class Meta:
+        model = User
+    first_name = fake.first_name()
+    email = factory.LazyAttribute(lambda n: "{}@example.com".format(
+        n.username))
+    username = factory.Sequence(lambda n: "user{}".format(n))
+
+
+class PhotoFactory(factory.Factory):
+    class Meta:
+        model = Photo
+
+
+class AlbumFactory(factory.Factory):
+    class Meta:
+        model = Album
+
+
+class PhotoTestCase(TestCase):
+
+    def setUp(self):
+        self.user = UserFactory.create()
+        self.user.save()
+        self.photo = PhotoFactory.create(user=self.user)
+
+    def tearDown(self):
```

Comment:
This should not be required.  Django enforces isolation between test cases by wiping out stuff you created in `setUp`